specification/TileFormats/Batched3DModel/README.md

```diff
@@ -65,72 +41,80 @@ The body section immediately follows the header section, and is composed of thre
 Contains values for `b3dm` semantics.
 More information is available in the [Feature Table specification](../FeatureTable/README.md).
 
-#### Property Reference
-
-* [`Extension`](#reference-extension)
-* [`Extras`](#reference-extras)
-* [`Batched 3D Model Feature Table`](#reference-feature-table) 
-
-
----------------------------------------
-<a name="reference-extension"></a>
-##### Extension
-
-Dictionary object with extension-specific objects.
-
-Additional properties are allowed.
-
-* **JSON schema**: [extension.schema.json](schema/extension.schema.json)
-
-
-
-
----------------------------------------
-<a name="reference-extras"></a>
-##### Extras
-
-Application-specific data.
+### Property Reference
```

Comment:
Organizationally, I think it makes the document easier to read if the property reference section is the last rather than somewhere in the middle. Same for any other READMEs.

Reply:
Makes sense, I'll make them an "Appendix" and add a link from the Header section.